src/test/java/dal/fatih/todoly/TodolyTest.java

```diff
@@ -197,68 +200,79 @@ public void shouldNotAllowDueDateToBeEqualToNow() throws URISyntaxException {
 
     @Test
     @DirtiesContext(methodMode = DirtiesContext.MethodMode.BEFORE_METHOD)
-    public void shouldListAllTasks() throws URISyntaxException {
+    public void shouldListAllTasks() throws URISyntaxException, JsonProcessingException {
         LocalDateTime dueDate = LocalDateTime.now().plusMinutes(10);
         String title = "Listed-title-of-task", description = "Listed-description-of-task";
         DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm:ss");
-
+        //Created three tasks
         createTask(title, description, dueDate, null, null, null);

```

Comment:
Keep `createTask` method simple. Make it create only one task. You can then call it three times in this test for example to create three tasks. Always try to keep the methods as simple as possible. Otherwise, it might be hard to re-use the same method for other scenarios.